modules/light-clients/08-wasm/internal/ibcwasm/wasm.go

```diff
@@ -1,14 +1,21 @@
 package ibcwasm
 
 import (
-	storetypes "cosmossdk.io/store/types"
+	"cosmossdk.io/collections"
+	storetypes "cosmossdk.io/core/store"
 )
 
 var (
 	vm WasmEngine
-	// storeKeyMap stores the storeKey for the 08-wasm module. Using a global storeKey is required since
-	// the client state interface functions do not have access to the keeper.
-	wasmStoreKey storetypes.StoreKey
+
+	// state management
+	Schema     collections.Schema
+	CodeHashes collections.KeySet[[]byte]
```

Comment:
should these be private with getter functions? I'm okay with the `ibcwasm.CodeHashes`, but it does go against what pattern was used before. What's the benefit of the exposed Schema?

Reply:
This is the pattern used in the sdk keepers. They use glabal collections and also expose Schema (even though most of them are not used anywhere). Schema includes info about the state of the module, such as maps to collections based on their name or prefix. I'm guessing it is for external modules to use but I don't know why SDK does this?

Reply:
Here is an example: https://github.com/cosmos/cosmos-sdk/blob/393de266c8675dc16cc037c1a15011b1e990975f/x/bank/keeper/view.go#L57-L93

Reply:
would probs lobby for unexported vars and getters which expose access to them 👍

Reply:
Leave this to a follow up as I think we need more consensus on this one